### PR TITLE
chore: add Typescript getting started demo

### PR DESCRIPTION
**Pull request by PekoPPT**

### CODE REVIEW

File: getting-started-app-typescript/src/App.vue [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        this.filter = {
+          logic: 'and',
+          filters: [{ field: 'CategoryID', operator: 'eq', value: e.target.value.CategoryID }]
+        }
+        this.skip = 0
+      } else {
+        this.filter = {} as State["filter"]
+        this.skip = 0
+      }
+      let event: GridDataStateChangeEvent = {
```
Comment (zdravkov) [inline getting-started-app-typescript/src/App.vue:113]: better a const

File: getting-started-app-typescript/src/App.vue [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+      defaultItems: { CategoryID: null, CategoryName: "Product categories" },
+      dropdownlistCategory: null,
+      pageable: true,
+      sortable: true,
+      skip: 0 as State["skip"],
+      take: 10 as State["take"],
+      sort: [
+        { field: "ProductName", dir: "asc" }
+      ] as State["sort"],
+      filter: {} as State["filter"],
```
Comment (zdravkov) [inline getting-started-app-typescript/src/App.vue:78]: we can use CompositeFilterDescriptor here

File: getting-started-app-typescript/src/App.vue [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+  },
+  data: function () {
+    return {
+      categories: categories,
+      products: products,
+      defaultItems: { CategoryID: null, CategoryName: "Product categories" },
+      dropdownlistCategory: null,
+      pageable: true,
+      sortable: true,
+      skip: 0 as State["skip"],
```
Comment (zdravkov) [inline getting-started-app-typescript/src/App.vue:73]: State['skip'] and take are just ?number

File: getting-started-app-typescript/src/App.vue [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+      this.dropdownlistCategory = e.target.value.CategoryID;
+
+      if (e.target.value.CategoryID !== null) {
+        this.filter = {
+          logic: 'and',
+          filters: [{ field: 'CategoryID', operator: 'eq', value: e.target.value.CategoryID }]
+        }
+        this.skip = 0
+      } else {
+        this.filter = {} as State["filter"]
```
Comment (zdravkov) [inline getting-started-app-typescript/src/App.vue:110]: same CompositeFilterDescriptor

File: getting-started-app-typescript/src/App.vue (hunk 1 above)
Comment (PekoPPT) [reply getting-started-app-typescript/src/App.vue:113]: fixed

**Pull request merged**